dcs/core/models.py

```diff
@@ -54,6 +54,7 @@ class DrsObject(BaseModel):
     """
 
     file_id: str
+    object_id: str
```

Comment:
Maybe we can add another DrsObjectCreation Model that does not include the object ID because it is chosen by the service on registration time.